docs/stdlib/type.rst

```diff
@@ -231,10 +231,11 @@ Types
         with module example
         select Text {
             name :=
-                Text[is Issue].name IF Text is Issue ELSE
+                Text[is Issue].name if Text is Issue else
                 <str>{},
                 # the cast to str is necessary here, because
-                # the type of the computed expression must be defined
+                # the type of the computed expression must be
+                # defined
```

Comment:
I changed the word wrap location here so that the comment would be fully visible without needing to horizontally scroll the code example when viewing the page in a medium-width browser.